File: answers/NodeJS/@CFCIfe/01-name-concatenation.js
const readlineSync = require("readline-sync");
function nameConcatenation() {
  let firstName = readlineSync.question("First Name: "),
    lastName = readlineSync.question("last name: "),
    yearOfBirth = readlineSync.questionInt("Year of birth: "),
    gender = readlineSync.question("binary gender (M/F): "),
    today = new Date(),
    currentYear = today.getFullYear(),
    age = currentYear - yearOfBirth;

  return !isNaN(age)
    ? `Welcome, ${firstName.toUpperCase()} ${
        gender.toLowerCase() == "m"
          ? "Son"
          : gender.toLowerCase() == "f"
          ? "Daughter"
          : ""
      } of  ${lastName.toUpperCase()}  ${age} years old`
    : " ";
}

console.log(nameConcatenation());

File: programming/answers/Python/@oseme-techguy/01-name-concatenation.py
"""
    Solution to Name Concatenation
"""

if __name__ == '__main__':
    first_name = input('Enter your first name: ')
    first_name = str(first_name)

    last_name = input('Enter your last name: ')
    last_name = str(last_name)

    age = input('Enter your age: ')
    age = int(age)

    print('\nWelcome, {first_name} {last_name} ({age})\n'.format(
        first_name=first_name,
        last_name=last_name,
        age=age
    ))
